Java

```diff
@@ -1,44 +1,55 @@
 import java.util.Scanner; 
-import java.text.NumberFormat;
-class student{   
```

Comment:
Since the name of the file needs to match the name of the class, unfortunately the initial file did not compile.